### ISSUE TITLE
How can i solve the issue - The application is not configured properly (message consumers are not available). Please contact the system administrator.

### ISSUE THREAD

**Issue by Jeeva-Rathinam** [oroinc/crm#316]
After installing just logged out and logged in i am getting an error like 

**The application is not configured properly (message consumers are not available). Please contact the system administrator.**

How to solve the notification.

**Comment by anyt** [comment]
Message queue consumer must be enabled.
Please follow the installation documentation
https://oroinc.com/orocrm/doc/2.6/install-upgrade/installation-quick-start-dev/crm#configure-and-run-required-background-processes

**Comment by vytsci** [comment]
You must configure Supervisor to run oro:message-queue:consume command, but be careful because add --no-debug parameter as it will drive your hard drive crazy.

**Comment by Jeeva-Rathinam** [comment]
> Message queue consumer must be enabled.
> Please follow the installation documentation
> https://oroinc.com/orocrm/doc/2.6/install-upgrade/installation-quick-start-dev/crm#configure-and-run-required-background-processes

yes, it is configured, still not sync, getting error like **The application is not configured properly (message consumers are not available). Please contact the system administrator.**

**Comment by Jeeva-Rathinam** [comment]
> Make sure you have run the message consumer and an application in the same environment. If it is, try to restart the consumer and check the consumer logs for errors.

yes both are in same environment.
